config/monitoring/networkpolicy/operator/operator.yaml

```diff
@@ -22,5 +22,8 @@ spec:
         - namespaceSelector:
             matchLabels:
               opendatahub.io/generated-namespace: "true"
+        - namespaceSelector:
```

Comment:
hum, so any downstream offering (self and managed) will have this problem because operator webhook will try to talk to host network? (is that a lot)?

Reply:
invite @lburgazzoli to the party

Reply:
It's about api-server to talk to webhook

Reply:
so the kube-api is in the host-network, and sending admission back to webhook :thinking: 
then i think it makes sense

Reply:
yep